custom_dc/index.md

```diff
@@ -95,3 +95,13 @@ The cost of running a site on Google Cloud Platform depends on the size of your
 1. Build a custom image, upload it to the Google Cloud Artifact Registry and create a Cloud Run service to run the site. See [Deploy services to Google Cloud](deploy_cloud.md)
 1. Launch and productionize your site for external traffic. See [Launch your Data Commons](/custom_dc/launch_cloud.html).
 1. For future updates and launches, continue to make UI and data changes locally, before deploying the changes to GCP.
+
+## Send feedback {#feedback}
+
+We use [https://issuetracker.google.com](https://issuetracker.google.com){: target="_blank"} to track bugs and feature requests. All tickets are is publicly viewable by all users and partners of Data Commons.
+
+- To make a feature request, please see if an [existing request]https://issuetracker.google.com/issues?q=componentid:1659535%2B%20type:feature_request){: target="_blank"} already exists. If it does, upvote and subscribe to it. If not, open a new feature request[here](https://issuetracker.google.com/issues/new?component=1659535){: target="_blank"}.
```

Comment:
Can we include a screenshot that indicates the buttons for upvoting and subscribing?

Reply:
Let's not use screenshots, but how about I just add text to describe how?

Reply:
Sounds good!

Reply:
Done.

Reply:
Reopening this thread to ask if we can mention that you can subscribe to an issue by starring it. Alternatively we could link to this documentation: https://developers.google.com/issue-tracker/guides/subscribe

Reply:
Yes, that's great. PTAL.